examples/wms-custom-tilegrid-512x256.js

```diff
@@ -25,7 +25,6 @@ var layers = [
     source: new ol.source.MapQuest({layer: 'sat'})
   }),
   new ol.layer.Tile({
-    extent: [-13884991, 2870341, -7455066, 6338219],
```

Comment:
Exactly. See https://github.com/openlayers/ol3/issues/2091#issuecomment-43783362 for the reasoning. If this turns out useful, we could later add the extent as source option and pass it to the tile grid.